### ISSUE TITLE
no_cursor_warps=true prevents switching to empty workspace on different monitor

### ISSUE THREAD

**Issue by Meikk99** [hyprwm/Hyprland#628]
On a setup with 2 monitors it is impossible to switch to the workspace on the second monitor if there are no windows on it and the mouse is on the first monitor, this occurs only when `no_cursor_warps=true` is set in the config.

How to reproduce:
set `no_cursor_warps=true`
without opening any windows on the second monitor try switching to it (bind=SUPER,1,workspace,2 or hyprctl dispatch workspace 2)
result: it only works if either the mouse is on monitor 2 or a window is open on workspace 2

I considered this might also have to do with `follow_mouse`, but regardless of what that is set to the exact same behavior occurs. 

My config:
```
monitor=DP-1,preferred,1920x0,1
monitor=DP-2,preferred,0x0,1

workspace=DP-1,1
workspace=DP-2,2

input {
    follow_mouse=0
}

general {
    main_mod=SUPER
    damage_tracking=full
    no_cursor_warps=true
}

# keybinds
bind=SUPER,Q,killactive
bind=SUPER,RETURN,exec,foot

bind=SUPER,h,movefocus,l
bind=SUPER,l,movefocus,r
bind=SUPER,k,movefocus,u
bind=SUPER,j,movefocus,d

bind=SUPERSHIFT,h,movewindow,l
bind=SUPERSHIFT,l,movewindow,r
bind=SUPERSHIFT,k,movewindow,u
bind=SUPERSHIFT,j,movewindow,d

bind=SUPER,1,workspace,1
bind=SUPER,2,workspace,2

bind=SUPERSHIFT,1,movetoworkspacesilent,1
bind=SUPERSHIFT,2,movetoworkspacesilent,2

```

**Comment by vaxerski** [comment]
check with 94ca386a8c298265fa5fb19ee2ec2be3db335502, should be fixed

**Comment by Meikk99** [comment]
Thank you, as far as I can tell this issue is fixed. Though it made me notice that new windows are always placed on the workspace containing the cursor, rather than the active workspace.

**Comment by vaxerski** [comment]
fixed in c1bc8d46e971b218b43bcbc31f0a854a5df5c0c9
